### PR TITLE
Signups report

### LINKED ISSUE #179: generate signups report for admins
generate and format a report for admins listing all signups by date for a given date range

### PR DESCRIPTION
**Pull request by ilyanaDev**
fixes #179

### CODE REVIEW

File: src/DevBetterWeb.Core/Entities/BillingActivity.cs [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ﻿using System;
+using DevBetterWeb.Core.Interfaces;
 using DevBetterWeb.Core.SharedKernel;
 using DevBetterWeb.Core.ValueObjects;
 
 namespace DevBetterWeb.Core.Entities
 {
-  public class BillingActivity : BaseEntity
+  public class BillingActivity : BaseEntity, IAggregateRoot
```
Comment (ardalis) [inline src/DevBetterWeb.Core/Entities/BillingActivity.cs:8]: I wonder if BillingActivity makes sense as an aggregate root, or if it should simply be a collection on Member, which is already an Aggregate Root. But, I think we probably aren't using aggregates effectively yet in this solution so I'll probably let this go for this PR.

File: src/DevBetterWeb.Core/Interfaces/ICSVService.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using DevBetterWeb.Core.Entities;
+
+namespace DevBetterWeb.Core.Interfaces
+{
+  public interface ICSVService
+  {
+    byte[] GetCSVByteArrayFromList(List<BillingActivity> billingActivities);
```
Comment (ardalis) [inline src/DevBetterWeb.Core/Interfaces/ICSVService.cs:8]: Renam CSV to Csv throughout.
`ICSVService` => `ICsvService`
etc.

**Review by ardalis — CHANGES_REQUESTED** [review]
Overall looks good. I assume I probably need to take your other data schema change PR before this one. One minor rename change.

**Review by ardalis — COMMENTED** [review]
Don't forget CSVService =-> CsvService

**Review by ardalis — APPROVED** [review]

**Comment by ilyanaDev** [comment]
@ardalis  can this be merged?

**Comment by ardalis** [comment]
Hopefully this doesn't break anything...

**Pull request merged**